common/static/css/edx-cc.css

```diff
@@ -0,0 +1,63 @@
+@font-face {
+  font-family: 'edx-cc';
+  src: url('../fonts/edx-cc/edx-cc.eot?52318265');
+  src: url('../fonts/edx-cc/edx-cc.eot?52318265#iefix') format('embedded-opentype'),
+       url('../fonts/edx-cc/edx-cc.woff?52318265') format('woff'),
+       url('../fonts/edx-cc/edx-cc.ttf?52318265') format('truetype'),
+       url('../fonts/edx-cc/edx-cc.svg?52318265#edx-cc') format('svg');
+  font-weight: normal;
+  font-style: normal;
+}
+/* Chrome hack: SVG is rendered more smooth in Windozze. 100% magic, uncomment if you need it. */
+/* Note, that will break hinting! In other OS-es font will be not as sharp as it could be */
+/*
+@media screen and (-webkit-min-device-pixel-ratio:0) {
+  @font-face {
+    font-family: 'edx-cc';
+    src: url('../fonts/edx-cc/edx-cc.svg?52318265#edx-cc') format('svg');
+  }
+}
+*/
+
+ [class^="icon-cc"]:before, [class*=" icon-cc"]:before {
+  font-family: "edx-cc";
+  font-style: normal;
+  font-weight: normal;
+  speak: none;
+
+  display: inline-block;
+  text-decoration: inherit;
+  width: 1em;
+  margin-right: .2em;
```

Comment:
@marcotuts, I grabbed this css file from #6228, and I don't know if @esbeka wrote them or if he copied them from somewhere else. They probably need a lot of cleaning up. There's also a css file that specifically targets IE7, which we can probably drop since we don't support IE7. Do you want to review it on a higher level than altering specific lines in this file?